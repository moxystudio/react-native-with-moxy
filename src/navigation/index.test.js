import React from 'react';
import Navigation from './';
import { shallow } from 'enzyme';

it('should create navigation correctly', () => {
    const tree = shallow(<Navigation />);
    const ProfileScreen = tree.find('StackNavigator').children().find('[name="Profile"]');

    expect(ProfileScreen.props().options.header()).toEqual(null);

    expect(tree).toMatchSnapshot();
});
