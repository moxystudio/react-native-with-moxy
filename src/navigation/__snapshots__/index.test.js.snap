// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create navigation correctly 1`] = `
<StackNavigator>
  <Screen
    component={[Function]}
    name="Home"
  />
  <Screen
    component={[Function]}
    name="Profile"
    options={
      Object {
        "header": [Function],
      }
    }
  />
</StackNavigator>
`;
